### DISCUSSION TITLE
yolo achievement

### DISCUSSION THREAD

**Discussion by MohitBansal321** [community/community#50157] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My one of pr is merged without any review but didn't get yolo achievement yet.

**Comment by stress-tess** [comment, marked as answer]
You need to have someone assigned to review and then merge before they can review

**Comment by Anmol-Baranwal** [comment]
The "Yolo" achievement in GitHub is awarded when:
you submit a PR in a public repository and just either 
- directly merge it or 
- add a friend as a reviewer but merge it without your friend approving the PR.

You can read more [here](https://github.com/Schweinepriester/github-profile-achievements)

**Comment by mkstix6** [comment]
The PR my 'YOLO' award links to had 4 approvals before being merged 🤷‍♂️ .
Is it a bug? Or am I misunderstanding?
